libs/repositories/src/const.ts

```diff
@@ -0,0 +1,30 @@
+import BigNumber from 'bignumber.js';
+import { SupportedChainId } from './types';
+
+interface TokenAddressAndDecimals {
+  address: string;
+  decimals: number;
+}
+
+export const USDC: Record<SupportedChainId, TokenAddressAndDecimals> = {
+  [SupportedChainId.MAINNET]: {
+    address: '0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48',
+    decimals: 6,
+  },
+  [SupportedChainId.GNOSIS_CHAIN]: {
+    address: '0x2a22f9c3b484c3629090feed35f17ff8f88f76f0', // '0xDDAfbb505ad214D7b80b1f830fcCc89B60fb7A83',
```

Comment:
Why the second commented address?

Reply:
One is USDe and the one in the comment USD. I will delete the comment

Reply:
I removed it, i don't think we need that there